Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q -f \
+		./*.md doc src lib benchmark tools/doc/ tools/icu/
```

Comment:
Not sure about the default, but we can't use just `./`, because we want to exclude other thirdparty modules that we have in the `./tools/` directory and the `./deps/` directory.

`./*.md doc src lib benchmark tools/doc/ tools/icu/` is unfolded to `./BUILDING.md ./CHANGELOG.md ./CODE_OF_CONDUCT.md ./COLLABORATOR_GUIDE.md ./CONTRIBUTING.md ./GOVERNANCE.md ./README.md doc src lib benchmark tools/doc/ tools/icu/` by the shell.

This exclicitly lists everything except for the directories in `tools/` and `deps/` that we want to exclude. It also doesn't include `.github/`, as that has templates which are exceptions from some rules.